### ISSUE TITLE
Copter: wiki updates for AC3.5

### ISSUE THREAD

**Issue by rmackay9** [ArduPilot/ardupilot#5160]
The list of wiki page updates that should be completed before AC3.5 is released:
- [x] flight time recorder: http://ardupilot.org/copter/docs/common-flight-time-recorder.html
- [x] servo gripper: http://ardupilot.org/copter/docs/common-gripper-servo.html
- [x] how to specifying sensor offsets for EKF (GPS, compass). http://ardupilot.org/copter/docs/common-sensor-offset-compensation.html
- [ ] SF40c setup instructions (setting yaw-offset, ignore zones)
- [ ] RNGFND_TYPE needs to change for Maxbotix I2C sonar (2), LidarLite (3) and TeraRange One (14)
- [x] pixart optical flow sensor
- [ ] simulataneous optical flow and gps (https://github.com/ArduPilot/ardupilot/issues/5484)
- [x] how to move motors around by changing SERVOX_FUNCTION http://ardupilot.org/antennatracker/index.html
- [x] tricopter MOT_YAW_SV_MIN, MAX, TRIM now set with SERVO7_MIN, MAX, TRIM. http://ardupilot.org/copter/docs/tricopter.html
- [x] precision loiter setup on auxiliar switch and IRLock page (http://ardupilot.org/copter/docs/precision-landing-with-irlock.html)
- [x] set FRAME_CLASS parameter. http://ardupilot.org/copter/docs/frame-type-configuration.html
- [x] package-place on mission command list page (http://ardupilot.org/copter/docs/mission-command-list.html)
- [x] UBlox M8P RTK GPS support (aka Here+)  http://ardupilot.org/copter/docs/common-here-plus-gps.html
- [x] maxell and solo smart battery support. http://ardupilot.org/copter/docs/common-smart-battery.html
- [ ] hexa, octa H frame description including images (http://ardupilot.org/copter/docs/connect-escs-and-motors.html#hexa-octo-y6)
- [ ] add BEC and wiring description for RFD900
- [x] add BeagleBoneBlue to list of supported autopilots. http://ardupilot.org/copter/docs/common-beagle-bone-blue.html
- [x] singlecopter / coaxcopter motor order changes: http://ardupilot.org/copter/docs/singlecopter-and-coaxcopter.html
- [x] Pixsi Multi GPS should be added to GPS page.  http://ardupilot.org/copter/docs/common-piksi-multi-rtk-receiver.html
- [x] Dual GPS Blending page.  http://ardupilot.org/copter/docs/common-gps-blending.html
- [x] telemetry setup page mentions MAVLink2. http://ardupilot.org/copter/docs/common-telemetry-port-setup-for-apm-px4-and-pixhawk.html
- [x] auxiliary function switch options need adsb-avoidance, object avoidance (http://ardupilot.org/copter/docs/channel-7-and-8-options.html)
- [x] autotune using position hold, and add "common problems" section to mention the pilot-override problem. http://ardupilot.org/copter/docs/autotune.html
- [x] object avoidance using simple lidar. http://discuss.ardupilot.org/t/copter-object-avoidance-with-upward-facing-lidar
- [ ] improve LidarLiteV3 page especially pwm wiring diagram. http://ardupilot.org/copter/docs/common-rangefinder-lidarlite.html
- [x] ekf type and imu mask. http://ardupilot.org/copter/docs/common-apm-navigation-extended-kalman-filter-overview.html
- [x] many pages need change from RCx_FUNCTION to SERVOx_FUNCTION (i.e. gripper)
- [ ] dataflash log pages need column name updates
- [x] update buzzer to refer to modern (post-APM) buzzers: http://ardupilot.org/copter/docs/nav-leds.html
- [ ] update prearm checks page: http://ardupilot.org/copter/docs/prearm_safety_check.html

Updates for AC3.4 can be seen here: https://github.com/ArduPilot/ardupilot/issues/4674

**Comment by magicrub** [comment]
Leddarone lidar

**Comment by magicrub** [comment]
Btw, wouldn't this be best done as an issue in the wiki issue trackerr?

**Comment by rmackay9** [comment]
Leddar one is actually already done for AC3.4: http://ardupilot.org/copter/docs/common-leddar-one-lidar.html

I like having this item here because it's easier to include (I think) with our code issues which are the main drivers for the release.

**Comment by magicrub** [comment]
I'll update the release version on that page to v3.5

**Comment by OXINARF** [comment]
@magicrub LeddarOne will be in 3.4.2

**Comment by magicrub** [comment]
Yeah, that's what I meant 😀

**Comment by magicrub** [comment]
wiki LeddarOne updated to v3.4.2: https://github.com/ArduPilot/ardupilot_wiki/commit/04a799c96352b1736f34bc99f77e701dfb9e8236?w=1

**Comment by patrickpoirier51** [comment]
Suggestion:
Update the optical flow page by adding the Garmin Lidar Lite v3  (I2C mode)
Remove PulseLight because this product no longer exist.,
Keep a reference for pulselight  name  because Lidar Lite is the subsitution 
Option: Renaming pulselight  to lidarlite on the code ?

**Comment by RQRobert** [comment]
Kindly update

"dodeca-hexa esc wiring and propeller direction in 

Connect ESCs and Motors Wiki Page

**Comment by tonysmartantzoulatos** [comment]
Is there a way to set flight time recorder to produce a warning message on MP if a certain flight time has been reached in order to maintain periodic checks on vehicles?

**Comment by rmackay9** [comment]
@tonysmartantzoulatos, it's possible to add that but it's not a feature we have at the moment.  Feel free to raise a new enhancement request although my feeling is that this should be a feature of a ground based drone maintenance application.. i.e. a fancy GCS or server based system that keeps an eye on all the drones a person or company owns and automatically analyses logs looking for problems.  I haven't seen too many of these types of programs but I suspect they exist.

**Comment by rmackay9** [comment]
I think I'll close this now because it's quite old.  We are maintaining wiki to-do items on the wiki issues list now: https://github.com/ArduPilot/ardupilot_wiki/issues
